src/base/mem_block.hpp

```diff
@@ -306,6 +306,8 @@ namespace cubmem
   extensible_block::extensible_block (extensible_block &&b)
     : extensible_block { *b.m_allocator }
   {
+    m_block.dim = b.m_block.dim;
+    m_block.ptr = b.m_block.ptr;
     b.m_block.dim = 0;
     b.m_block.ptr = NULL;
```

Comment:
```suggestion
    m_block = std:move (b.m_block);
```
How about it to make use of the defend move op of `struct block`?

Reply:
However, `extensible_block::operator= (extensible_block &&b)` also define its own move. We can go with your opinion.

Reply:
Thank you. I'd like to accept this suggestion.